bigquery_etl/dryrun.py

```diff
@@ -88,7 +88,7 @@
     "sql/moz-fx-data-shared-prod/fivetran_costs_derived/incremental_mar_v1/query.sql",
     "sql/moz-fx-data-shared-prod/fivetran_costs_derived/monthly_costs_v1/query.sql",
     *glob.glob(
-        "sql/moz-fx-data-shared-prod/apple_ads/**/view.sql",
+        "sql/moz-fx-data-shared-prod/apple_ads_external/**/query.sql",
```

Comment:
This will not work until dryrun fix for staging CI step is merged. PR#3858 addresses this issue and will need to be merged prior to this PR.

Reply:
the above is no longer correct. Explanation of the issue can be found here:
https://github.com/mozilla/bigquery-etl/pull/3856#issuecomment-1564318101

The currently agreed upon solution is to wildcard the project_id in the skip list.